src/cli-validator/utils/printJsonResults.js

```diff
@@ -33,6 +33,10 @@ module.exports = function printJson(results, originalFile) {
     });
   });
   // render the results to json in the console with 2 char spacing
```

Comment:
The above comment goes over the line `const jsonstr = JSON.stringify(results, null, 2);`